styles.css

```diff
@@ -0,0 +1,252 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  display: flex;
+  height: 15rem;
+  padding: 5% 3% 3% 8%;
+  border: none;
+  justify-content: space-between;
+}
+
+h1 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-style: italic;
+  font-size: 1.5rem;
+  color: rgb(0, 179, 60);
+}
+
+h2 {
+  font-family: 'Ibarra Real Nova', serif;
+}
+
+h3 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-size: 1.5rem;
+}
+
+h4 {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.container {
+  width: 80%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.intro {
+  padding-right: 2rem;
+  font-size: 20px;
```

Comment:
Make sure to use rems for font-sizes. If you're basing off of 16px a 20px would be 1.25rem